README.md

```diff
@@ -128,9 +128,16 @@ The internal oidc provider gets that user's name, email and tenants from claims.
 
 The claim keys needed are set in the env variables:
 
-- INTERNAL_OIDC_CLAIM_NAME
-- INTERNAL_OIDC_CLAIM_EMAIL
-- INTERNAL_OIDC_CLAIM_GROUPS
+- `INTERNAL_OIDC_CLAIM_NAME`
+- `INTERNAL_OIDC_CLAIM_EMAIL`
+- `INTERNAL_OIDC_CLAIM_GROUPS`
+
+The value of the claim with the name that is defined in the env variable `INTERNAL_OIDC_CLAIM_GROUPS` is mapped to
+the user's access to tenants in `App\Security\AzureOidcAuthenticator`. The claim field should consist of an array of
+names that should follow the following structure `<TENANT_NAME><ROLE_IN_TENANT>`.
```

Comment:
Having a non-alphanumeric separator between `<TENANT_NAME>` and `<ROLE_IN_TENANT>` would probably be a good idea, e.g. `<TENANT_NAME>\<ROLE_IN_TENANT>`.

Reply:
Agreed. But that is not how it has been implemented.
The values for <ROLE_IN_TENANT> should be configurable:
https://github.com/os2display/display-api-service/blob/develop/src/Security/AzureOidcAuthenticator.php#L34